From 7914a8fb2bef50483a6778e15c4144d5c5760790 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 25 Dec 2020 11:37:10 +0800
Subject: [PATCH] arm64: dts: rockchip: Add arm,no-tick-in-suspend in timer for
 rk3568

Arch timer stops during system suspend. Add arm,no-tick-in-suspend
property in timer.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I24ebe7e027593cfbdbfc5476f196425df640e8d7
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index cc54c57db71a..1695d985153a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -275,6 +275,7 @@
 			     <GIC_PPI 14 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>,
 			     <GIC_PPI 11 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>,
 			     <GIC_PPI 10 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>;
+		arm,no-tick-in-suspend;
 	};
 
 	gmac0_clkin: external-gmac0-clock {
-- 
2.35.3

